ci-operator/step-registry/openshift-qe/installer/bm/day2/regulus/openshift-qe-installer-bm-day2-regulus-commands.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+LAB_CLOUD="${LAB_CLOUD:-}"
+BASTION="${BASTION:-}"
+
+if [ -z "${RUNLOCAL:-}" ]; then
+  bastion=$(cat "/bm/address")
+  SSH_ARGS="-i /bm/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+else
+  bastion=$BASTION
+  SSH_ARGS="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+fi
+
+regulus_repo="/root/REGULUS/regulus-${LAB_CLOUD}-$(date "+%Y-%m-%d-%H-%M-%S")"
+crucible_install="/root/crucible-internal"
+
+install-crucible() {
+  # Setup Bastion
+  ssh ${SSH_ARGS} root@${bastion} "
+    set -e
+    set -o pipefail
+
+    if [ -d  /opt/crucible ] ; then
+        rm -fr /opt/crucible
+    fi
+    if [ -d  ${crucible_install} ] ; then
+        rm -fr  ${crucible_install}
+    fi
+    git clone -c http.sslVerify=false https://gitlab.cee.redhat.com/atheurer/crucible-internal.git ${crucible_install}
```

Comment:
As we discussed in the CPT meetup please don't use private user repositories for the CPT pipelines, only repositories that are part of redhat organizations, i.e.: [redhat-performance](https://github.com/redhat-performance)

Reply:
Correct, we are using the same install method with jetlag. 

I am still sort of lost on why we need two different ways to install Crucible (one in jetlag, and one here).

Reply:
With respect to crucible, it seems we may have a few conciderations.
Issues:
=====
1. https://gitlab.cee.redhat.com/atheurer/crucible-internal.git exposes authorization tokens. Should we hide these tokes in secrets.
2. This repo is a private repo. Is it OK?

Not issues but options:
===============
1. Install crucible by jetlag. Pros: none. Cons: changing crucible installation requires rerun jetlag (1hr long process).  Also, do we have any "as" that we want jetlag without crucible?
2. Install crucible by its own "as" - Pros: efficiency. Cons: none
3. Install crucible by Regulus - Pros: crucible is a depdedency of Regulus only. So let regulus install it just like any dnf package that regulus needs. Cons:none

Reply:
I will use the assumption that jetlag will install crucible.